cli/args/lockfile.rs

```diff
@@ -192,7 +195,18 @@ impl CliLockfile {
             {
               let config = WorkspaceMemberConfig {
                 package_json_deps: pkg_json_deps(folder.pkg_json.as_deref()),
-                dependencies: deno_json_deps(folder.deno_json.as_deref()),
+                dependencies: deno_json_deps(folder.deno_json.as_deref())
+                  .inspect_err(|err| {
+                    log::warn!(
+                      "failed to read dependencies from {}: {err}",
+                      folder
+                        .deno_json
+                        .as_ref()
+                        .map(|s| s.specifier.as_str())
+                        .unwrap_or("config file")
+                    )
+                  })
+                  .unwrap_or_default(),
```

Comment:
if reading the external import map fails, should we error out, warn (as it is currently), or just ignore?

Reply:
I think we can get this from the Workspace IIRC because specifying an import map overrides all deno.json imports/exports. Then we wouldn’t have to load again. I may be wrong though